migrations/create_activity_log_table.php.stub

```diff
@@ -5,12 +5,24 @@ use Illuminate\Database\Migrations\Migration;
 
 class CreateActivityLogTable extends Migration
```

Comment:
Please remove all changes to the migration. It makes no sense to use the config variable here. People can just update the migration itself.